src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -43,6 +43,7 @@ The .NET Foundation licenses this file to you under the MIT license.
     <UseSystemResourceKeys Condition="$(IlcDisableReflection) == 'true'">true</UseSystemResourceKeys>
     <EventSourceSupport Condition="$(IlcDisableReflection) == 'true'">false</EventSourceSupport>
     <EventSourceSupport Condition="$(EventSourceSupport) == ''">false</EventSourceSupport>
+    <DynamicCodeSupport Condition="'$(DynamicCodeSupport)' == ''">false</DynamicCodeSupport>
```

Comment:
Is there any way to test these things in `dotnet/runtime`?

Reply:
We compile all NativeAOT tests in dotnet/runtime using this target so we could check for this in NativeAOT (look for the appcontext switch at runtime). But it won't cover CoreCLR so we could still have a bug that this is only set in Publish and not know about it. Not sure it's worth the trouble just for that.